libparsec/crates/protocol/tests/serialization.rs

```diff
@@ -0,0 +1,201 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+// TODO: Define and use a proc macro `parsec_protocol_serialization_tests`
+
+// use libparsec_serialization_format::parsec_protocol_serialization_tests
+// parsec_protocol_serialization_tests!("schema/invited_cmds");
+// parsec_protocol_serialization_tests!("schema/authenticated_cmds");
+// parsec_protocol_serialization_tests!("schema/anonymous_cmds");
+
+// * The macro should crawl the schemas defined in libparsec/protocol/schemas
+// * and expand into code that will generate tests for each major API version.
+// *
+// * The auto-generated tests functions will call concrete functions that will
+// * have to be implemented by a developer in a specific test layout.
+// * For example, the tests created for "block_create.json5" schema will call
+// * functions from the following to-be-implemented files:
+// * - protocol/tests/v3/<block_create.rs>
+// * - protocol/tests/v4/<block_create.rs>
+
+// * WIP: the following code shows what the macro could generate for API v3/v4.
+
+mod v3 {
+    use libparsec_tests_fixtures::prelude::*;
+
+    //    use libparsec_protocol::anonymous_cmds::v3 as anonymous_cmds;
+    //    use libparsec_protocol::authenticated_cmds::v3 as authenticated_cmds;
+    //    use libparsec_protocol::invited_cmds::v3 as invited_cmds;
+
+    // * The path attribute prefixes inline mod paths with current mod name
+    // #[path="./v3/block_create.rs"]
+    #[path = "./block_create.rs"]
```

Comment:
Why you need to specify the path here ?

Reply:
You are probably right, it seemed necessary when using the `./v3/block_create.rs` path, but not anymore with the `./block_create.rs`. 
I'll update the proc macro :+1:

Reply:
Done